docs/gettsim_objects/input_variables.rst

```diff
@@ -146,3 +140,15 @@ household.
 || _`y_pflichtbeitr_ab_40`  || Total years of mandat. contributions after || float       |
 ||                          || age 40                                     ||             |
 +---------------------------+---------------------------------------------+--------------+
+|| _`anwartschaftszeit`     || Months of unemployment insurance           || int         |
+||                          || contributions in the last 30 months        ||             |
++---------------------------+---------------------------------------------+--------------+
+|| _`arbeitssuchend`        || Dummy: looking for employment              || bool        |
+||                          ||                                            ||             |
++---------------------------+---------------------------------------------+--------------+
+|| _`arbeitsl_geld_m_vorj`  || Amount of months in which Arbeitslosengeld || int         |
+||                          || was received in previous year              ||             |
++---------------------------+---------------------------------------------+--------------+
+|| _`arbeitsl_geld_m_v2j`   || Amount of months in which Arbeitslosengeld || int         |
```

Comment:
Can we summarize `arbeitsl_geld_m_v2j` and `arbeitsl_geld_m_vorj` in one parameter? Isn't the relevant variable how long the individual already receives unemployment contribution?

Reply:
Yes, if the input variable is how long the person already receives ALG 1 we can combine them.